listeners.js

```diff
@@ -1,17 +1,19 @@
 var auth = require('./auth')
+var debug = require('debug')('expressa/'+ String(__filename).replace(/.*\//, '') )
 
 module.exports = function(api) {
 
 	// Add db interface for new collections immediately
 	api.addListener('changed', function setupCollections(req, collection, data) {
 		if (collection == 'collection') {
 			api.db[data._id] = api.dbTypes[data.storage](req.settings, data._id)
-			console.log('updated ' + data._id + ' collection storage')
+			debug('updated ' + data._id + ' collection storage')
 		}
 	})
 
 	api.addListener(['post', 'put'], function updatePassword(req, collection, data) {
 		if (collection == 'users' && data.password && data.password.length != 60 && data.password[0] != "$") {
+			debug('setting new password')
```

Comment:
Maybe mention hashing in this comment, since it's pretty important that the plaintext password field is replaced with a hashed version.